specification/migrate/resource-manager/Microsoft.OffAzure/readme.go.md

```diff
@@ -0,0 +1,34 @@
+## Go
+
+These settings apply only when `--go` is specified on the command line.
+
+```yaml $(go) && !$(track2)
+go:
+  license-header: MICROSOFT_MIT_NO_VERSION
+  clear-output-folder: true
+  namespace: migrationdiscovery
+```
+
+```yaml $(go) && $(track2)
+license-header: MICROSOFT_MIT_NO_VERSION
+module-name: sdk/resourcemanager/migrationdiscovery/armmigrationdiscovery
+module: github.com/Azure/azure-sdk-for-go/$(module-name)
+output-folder: $(go-sdk-folder)/$(module-name)
+azure-arm: true
+```
+
+### Go multi-api
+
+```yaml $(go) && $(multiapi)
+batch:
+  - tag: package-2020-01
```

Comment:
This tag doesn't match any existing tags in the `readme.md` file which is why no input files can be found.

The value must align with the tag you intend to build.

Comment applies to all `readme.go.md` files.

Reply:
Just curious, why did others threw no error ?

Reply:
Looks like they did. See [python build log](https://github.com/Azure/azure-rest-api-specs/pull/30705/checks?check_run_id=32276795935).

Reply:
@jhendrixMSFT Thanks for the quick help. Even after updating the packages name I'm still getting the same error.